templates/repo/commits_table.tmpl

```diff
@@ -8,7 +8,7 @@
 			{{.locale.Tr "repo.commits.no_commits" $.BaseBranch $.HeadBranch}} {{if .RefName}}({{.RefName}}){{end}}
 		{{end}}
 	</div>
-	<div class="commits-table-right">
+	<div class="commits-table-right gt-whitespace-nowrap">
```

Comment:
Could you please add a screenshot?
I have some trouble imagining what the change is.

Reply:
Not necessary anymore, the linked issue suffices.

Reply:
Done, you checked latter